Code/CIPLabeler/Wrap/pyCIPLabelWrapTest.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+
+from rdkit import Chem
+
+
+class TestCase(unittest.TestCase):
+
+  def testLabelAtomsList(self):
+    mol = Chem.MolFromSmiles("C[C@H](Cl)CC[C@H](Cl)C")
+    assert mol
+
+    atom1 = mol.GetAtomWithIdx(1)
+    atom5 = mol.GetAtomWithIdx(5)
+
+    assert atom1.HasProp("_CIPCode") == True
+    assert atom5.HasProp("_CIPCode") == True
+
+    atom1.ClearProp("_CIPCode")
+    atom5.ClearProp("_CIPCode")
+
+    Chem.rdCIPLabeler.AssignCIPLabels(mol, [1], None)
```

Comment:
Please also add a test that uses a `set()`.

Reply:
Done.

Reply:
Please also add one that calls `AssignCIPLabels(mol)` (i.e. with just a molecule)

Reply:
That is used in the other Python test file, pyCIPLabelsValidation.py. Should I also add one test with that in pyCIPLabelWrapTest.py?

Reply:
> That is used in the other Python test file, pyCIPLabelsValidation.py. Should I also add one test with that in pyCIPLabelWrapTest.py?

Good point. Sorry I didn't think about that.